README.md

```diff
@@ -305,89 +298,42 @@ if you have already generated the build files, set environment variables, and th
-mkdir build
-cd build
-# generate files
-# cmake will automatically detect what C compiler is used by system by default and will generate files for it
-cmake ..
-# compile files. Cmake would call compiler and linker to generate libs
-cmake --build .
-```
-
-> Note: The steps above would compile and generate the default output for azure-sdk-for-c which includes static libraries only. See section [CMake Options](#cmake-options)
-
-#### Visual Studio 2019
-
-Open project folder with Visual Studio. If vcpkg has been previously installed and set up like mentioned [above](#vcpkg). Everything will be ready to build.
-Right after opening project, Visual Studio will read cmake files and generate cache files automatically.
-
-### Linux
-
-#### vcpkg
-
-vcpkg can be used to download packages sources, headers and build libraries for whatever TRIPLET is set up (platform/architecture).
-vcpkg maintains any installed package inside its own folder, allowing to have multiple vcpkg folder with different dependencies installed on each. This is also great because you don't have to install dependencies globally on your system.
+If the CMake option _-DCMAKE_TOOLCHAIN_FILE=..._ is not defined to generate the project, the Azure SDK project will automatically get Vcpkg and link it to get the required dependencies. You can use the next environment variables to change this behavior:
 
-Use the following steps to install vcpkg and have it linked to CMake.
+<center>
 
-> **Note:** The Azure SDK is only officially supported against certain versions of vcpkg. Use the commit in [vcpkg-commit.txt](https://github.com/Azure/azure-sdk-for-c/blob/main/eng/vcpkg-commit.txt) to get a known working version.
-
-```bash
-# Clone vcpkg:
-git clone https://github.com/Microsoft/vcpkg.git
-# (consider this path as PATH_TO_VCPKG)
-cd vcpkg
-# Checkout the vcpkg commit from the vcpkg-commit.txt file (link above)
-git checkout <vcpkg commit>
-
-# build vcpkg
-./bootstrap-vcpkg.sh
-./vcpkg install --triplet x64-linux curl cmocka paho-mqtt
-export VCPKG_DEFAULT_TRIPLET=x64-linux
-export VCPKG_ROOT=PATH_TO_VCPKG #replace PATH_TO_VCPKG for where vcpkg is installed
-```
-
-#### Debian
+<table>
+<tr>
+<td>Environment Variable</td>
+<td>Description</td>
+</tr>
+<tr>
+<td>AZURE_SDK_DISABLE_AUTO_VCPKG</td>
+<td>When defined, Vcpkg won't be automatically cloned and linked. Use this setting, for example, if your dependencies are installed on the system and you don't need to get them.</td>
+</tr>
+<tr>
+<td>AZURE_SDK_VCPKG_COMMIT</td>
+<td>This variable can set the git commit id to be used when automatically cloning Vcpkg.</td>
+</tr>
+<tr>
+<td>VCPKG_ROOT</td>
+<td>Use this variable to set an existing Vcpkg folder from your system to be linked for building. Use this, for example, when working with Vcpkg classic mode, to switch between different Vcpkg folders.</td>
+</tr>
+</table>
 
-Alternatively, for Ubuntu 18.04 you can use:
+</center>
```

Comment:
One can use markdown to specify the table

```suggestion
| Environment Variable | Description | 
| ------------------------| ------------ | 
| AZURE_SDK_DISABLE_AUTO_VCPKG | When defined, Vcpkg won't be automatically cloned and linked. Use this setting, for example, if your dependencies are installed on the system and you don't need to get them. | 
| AZURE_SDK_VCPKG_COMMIT | This variable can set the git commit id to be used when automatically cloning Vcpkg. | 
| VCPKG_ROOT | Use this variable to set an existing Vcpkg folder from your system to be linked for building. Use this, for example, when working with Vcpkg classic mode, to switch between different Vcpkg folders. | 
```

Resulting output looks like: 

| Environment Variable | Description | 
| ------------------------| ------------ | 
| AZURE_SDK_DISABLE_AUTO_VCPKG | When defined, Vcpkg won't be automatically cloned and linked. Use this setting, for example, if your dependencies are installed on the system and you don't need to get them. | 
| AZURE_SDK_VCPKG_COMMIT | This variable can set the git commit id to be used when automatically cloning Vcpkg. | 
| VCPKG_ROOT | Use this variable to set an existing Vcpkg folder from your system to be linked for building. Use this, for example, when working with Vcpkg classic mode, to switch between different Vcpkg folders. |